bletl/parsing/blpro.py

```diff
@@ -407,38 +407,42 @@ def transform_into_filtertimeseries(
+            if fs.filter_type == "Intensity" and ("Biomass" in fs.filter_name or "BS" in fs.filter_name):
+                key = f"BS{int(fs.gain_1)}"
+                times = measurements.xs(filter_number, level="filterset")["time"].unstack()
+                values = measurements.xs(filter_number, level="filterset")["amp_ref_1"].unstack()
+            elif fs.filter_type in {"pH", "DO"} and not return_uncalibrated_optode_data:
+                key = fs.filter_type
+                times = measurements.xs(filter_number, level="filterset")["time"].unstack()
+                values = measurements.xs(filter_number, level="filterset")["cal"].unstack()
+            elif fs.filter_type in {"pH", "DO"} and return_uncalibrated_optode_data:
+                key = fs.filter_type
+                times = measurements.xs(filter_number, level="filterset")["time"].unstack()
+                values = measurements.xs(filter_number, level="filterset")["phase"].unstack()
+            elif fs.filter_type == "Intensity":
+                key = fs.filter_name
+                times = measurements.xs(filter_number, level="filterset")["time"].unstack()
+                values = measurements.xs(filter_number, level="filterset")["amp_ref_1"].unstack()
+            else:
+                logger.warn(
+                    f'Skipped {fs.filter_type} channel with name "{fs.filter_name}" because no processing routine is implemented.'
+                )
+                continue
+
+            # transform into nicely formatted DataFrames for FilterTimeSeries
+            times.columns = [no_to_id[c] for c in times.columns]
+            times = times.reindex(sorted(times.columns), axis=1)
+            times.columns.name = "well"
+            values.columns = [no_to_id[c] for c in values.columns]
+            values = values.reindex(sorted(values.columns), axis=1)
+            values.columns.name = "well"
+            fts = FilterTimeSeries(times, values)
+            yield (key, fts)
+        else:#if filterset is not available like in issue 24 print warning
             logger.warn(
-                f'Skipped {fs.filter_type} channel with name "{fs.filter_name}" because no processing routine is implemented.'
-            )
-            continue
-
-        # transform into nicely formatted DataFrames for FilterTimeSeries
-        times.columns = [no_to_id[c] for c in times.columns]
-        times = times.reindex(sorted(times.columns), axis=1)
-        times.columns.name = "well"
-        values.columns = [no_to_id[c] for c in values.columns]
-        values = values.reindex(sorted(values.columns), axis=1)
-        values.columns.name = "well"
-        fts = FilterTimeSeries(times, values)
-        yield (key, fts)
-
+                    f'Skipped {fs.filter_type} channel with name "{fs.filter_name}" because no valid measurements available.'
```

Comment:
```suggestion
                    'Skipped channel %s with name "%s" because no valid measurements are available.', fs.filter_type, fs.filter_name
```
I know you adapted this from my code, but f-strings in log lines are dangerous and `pylint` warnst against it..